src/test/java/seedu/address/logic/commands/ReminderCommandTest.java

```diff
@@ -0,0 +1,100 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.ReminderCommand.MESSAGE_REMINDER;
+import static seedu.address.testutil.TypicalAddressBook.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalMeetings.COMPUTING_CLUB;
+import static seedu.address.testutil.TypicalMeetings.CS2101;
+import static seedu.address.testutil.TypicalMeetings.CS2103T;
+import static seedu.address.testutil.TypicalMeetings.DANCE_CLUB;
+import static seedu.address.testutil.TypicalMeetings.NUSSU;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.meeting.ReminderDatePredicate;
+
+
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code RemindCommand}.
+ */
+public class ReminderCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        ReminderDatePredicate firstPredicate = new ReminderDatePredicate(5);
+        ReminderDatePredicate secondPredicate = new ReminderDatePredicate(6);
+
+        ReminderCommand remindFirstCommand = new ReminderCommand(firstPredicate);
+        ReminderCommand remindSecondCommand = new ReminderCommand(secondPredicate);
+
+        // same values return true
+        ReminderCommand copy = new ReminderCommand(firstPredicate);
+        assertTrue(remindFirstCommand.equals(copy));
+
+        // different types return false
+        assertFalse(remindFirstCommand.equals("5"));
+
+        // null -> returns false
+        assertFalse(remindFirstCommand.equals(null));
+
+        // different predicate returns false
+        assertFalse(remindFirstCommand.equals(remindSecondCommand));
+    }
+
+    // Returns zero meetings due to invalid days
+    @Test
+    public void execute_invalidRange_noMeetingFound() {
+        String expectedMessage = String.format(MESSAGE_REMINDER, 0, -1);
+        ReminderCommand command = new ReminderCommand(new ReminderDatePredicate(-1));
+        expectedModel.updateFilteredMeetingList(new ReminderDatePredicate(-1));
+        assertCommandSuccess(command, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredMeetingList());
```

Comment:
I don't think this line is necessary, since `assertCommandSuccess` already compares the equality of the `models`, which in turn already compares the equality of the `filteredMeetingLists` in the models. 

Whether the filtering is done properly by the `ReminderDatePredicate` should be tested separately in tests for `ReminderDatePredicate`

Reply:
Similarly for the other tests below.